qmail-qfilter.c

```diff
@@ -0,0 +1,332 @@
+/* This is free and unencumbered software released into the public * domain. */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/wait.h>
+#include <signal.h>
+#include <unistd.h>
+
+#ifndef TMPDIR
+#define TMPDIR "/tmp"
+#endif
+
+#ifndef BUFSIZE
+#define BUFSIZE 4096
+#endif
+
+#ifndef QMAIL_QUEUE
+#define QMAIL_QUEUE "/var/qmail/bin/qmail-queue"
```

Comment:
If we provide it in-tree this must honor auto_qmail.

Reply:
Good call, fixed.